### PR TITLE
feat: add new ubuntu-18.04-amd64 config

### PR DESCRIPTION
**Pull request by tonyespy**
Add a new amd64 Ubuntu 18.04 config for snap builds.
This is the second step required to resolve issue #619.

Signed-off-by: Tony Espy <espy@canonical.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/edgexfoundry/ci-management/blob/master/.github/CONTRIBUTING.md
- [NA] Tests for the changes have been added (for bug fixes / features)
- [NA] Added labels

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Are there any new imports or modules? If so, what are they used for and why?
No

### CODE REVIEW

**Review by eb-oss — APPROVED** [review]

File: jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg
Comment (eb-oss) [inline jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg]: This should be IMAGE_NAME

**Review by eb-oss — CHANGES_REQUESTED** [review]

File: jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg
Comment (tonyespy) [reply jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg]: Thanks, I also realized it should use `=` too, not `:`. Just re-pushed.

Comment (eb-oss) [reply jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg]: You seem to have re-pushed without including the fix...

Comment (tonyespy) [reply jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg]: @#$&*! sorry about that, re-pushed, and verified this time.

File: jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg
Comment (ernestojeda) [inline jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg]: @tonyespy I think this should be changed to use the v2 High CPU flavor node. I did a bunch of testing on x86 nodes and found this to be the most performant for Snaps.

The change would look like this:
```
HARDWARE_ID=v2-highcpu-8
VOLUME_SIZE=40
```

Comment (tonyespy) [reply jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg]: @ernestojeda thanks for feedback, updated and re-pushed. Is the name of the file still OK with this change?

Comment (eb-oss) [reply jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg]: For a v2-highcpu-8, the file named should be changed to "8c-8g".

**Review by eb-oss — APPROVED** [review]
I'm okay with this, but I'll let @ernestojeda take a look and approve before merging.

File: jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg
Comment (ernestojeda) [reply jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg]: @tonyespy I created this node on the sandbox and then created a simple pipeline to run a test and it seems to work. https://jenkins.edgexfoundry.org/sandbox/view/All/job/Functional-Testing/job/ubuntu-node-test/2/console. Node label is `ubuntu18.04-docker-8c-8g`

**Review by ernestojeda — APPROVED** [review]
LGTM

**Pull request merged**
